code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { DETAILS_URL, IMAGE_URL, BACKGROUND_URL } from '../components/URLS'
+
+import '../styles/moviedetails.css'
+
+export const MovieDetails = () => {
+  const [details, setDetails] = useState({})
+  const { movieId } = useParams()
+
+  useEffect(() => {
+    fetch(DETAILS_URL(movieId))
+      .then((res) => res.json())
+      .then((data) => {
+        setDetails(data)
+      })
+  }, [movieId])
+
+  const generateRateClass = () => {
+    if (details.vote_average < 3) {
+      return 'details__low-rating'
+    } else if (details.vote_average < 7) {
+      return 'details__medium-rating'
+    } return 'details__high-rating'
+  }
```

Comment:
I like the idea of different colors for different ratings. As the css of these classes are commented out, these codes become obsolete. So I suggest either these lines of code need to be removed or the styling codes of these classes needs to be active.

Reply:
Thanks! I had forgotten that I commented out the styling for this function 🤣